dict/portuguese.yml

```diff
@@ -2,290 +2,290 @@
-    'NG': [Nigéria, República Federal da Nigéria, Nigerian*, Abuja, Lagos]
+    'MR': [Mauritânia, Mauritan*, Nuaquexote, Nuaquechote]
+    'NE': [Níger, Nigerin*, Nigerenseé*, Niamei, Niamey]
+    'NG': [Nigéria, Nigerian*, Abuja, Lagos]
     'SH': [Santa Helena, Santo, Santa-Helenense*, Jamestown, Cidade de Jamestown]
-    'SL': [Serra Leoa, República da Serra Leoa, Serra-Leonês*, Serra-Leonin*, Freetown]
-    'SN': [Senegal, República do Senegal, Senegalês*, Senegales*, Dacar, Dakar]
-    'TG': [Togo, República Togolesa, Togolês*, Lomé]
+    'SL': [Serra Leoa, Serra-Leonês*, Serra-Leonin*, Freetown]
+    'SN': [Senegal, Senegalês*, Senegales*, Dacar, Dakar]
+    'TG': [Togo, Togolês*, Togoles*, Lomé]
 
 AMERICA:
   CARIB:
     'AG': [Antígua e Barbuda, Antiga e Barbuda, Antiguan*]
     'AI': [Anguila, Anguilan*, The Valley]
     'AW': [Aruba, Aruban*, Oranjestad]
     'BB': [Barbados, Barbadian*, Barbadense*, Bridgetown]
-    'BL': [São Bartolomeu, Coletividade de São Bartolomeu, São-Bartolomense*, Gustávia]
+    'BL': [São Bartolomeu, São-Bartolomense*, Gustávia]
     'BQ': [Bonaire, Bonairense*, Kralendijk]
-    'BS': [Bahamas, Baamas, Comunidade das Bahamas, Bahamense*, Baamense*, Bahamian*, Baamian*, Bahamês*, Baamês*, Nassau]
-    'CU': [Cuba, República de Cuba, Cuban*, Havana]
+    'BS': [Bahamas, Baamas, Bahamense*, Baamense*, Bahamian*, Baamian*, Bahamês*, Baamês*, Nassau]
+    'CU': [Cuba, Cuban*, Havana]
     'CW': [Curaçau, Curaçauense*, Willemstad]
-    'DM': [Dominica, Domínica, Comunidade da Dominica, Dominiquense*, Dominiquês*, Dominicense*, Roseau]
+    'DM': [Dominica, Domínica, Dominiquense*, Dominiquês*, Dominicense*, Roseau]
     'DO': [República Dominicana, Dominican*, São Domingos]
     'GD': [Granada, Granadin*, São Jorge]
     'GP': [Guadalupe, Guadalupense*, Basse-Terre]
-    'HT': [Haiti, República do Haiti, Haitian*, Porto Príncipe, Porto do Príncipe]
+    'HT': [Haiti, Haitian*, Porto Príncipe, Porto do Príncipe]
     'JM': [Jamaica, Jamaican*, Kingston]
-    'KN': [São Cristóvão e Neves, São Cristóvão e Névis, Federação de São Cristóvão e Neves, Federação de São Cristóvão e Nevis, São-Cristovense, Basseterre]
+    'KN': [São Cristóvão e Neves, São Cristóvão e Névis, São-Cristovense, Basseterre]
     'KY': [Ilhas Cayman, Ilhas Caimã, Cayman, Caimão, Caimanês*, Caimanes*]
     'LC': [Santa Lúcia, Santa-Lucense*, Santa-Luciense*, Castries]
-    'MF': [São Martinho, Coletividade de São Martinho, São-Martinhense*, Marigot]
+    'MF': [São Martinho, São-Martinhense*, Marigot]
     'MQ': [Martinica, Martinican*]
     'MS': [Monserrate, Monserratense*, Brades, Brades Estate]
-    'PR': [Porto Rico, Estado Livre Associado de Porto Rico, Porto-Riquenh*, Porto-Riquense*, San Juan, São João, Município da Cidade Capital São João Batista]
+    'PR': [Porto Rico, Porto-Riquenh*, Porto-Riquense*, San Juan, São João, Município da Cidade Capital São João Batista]
```

Comment:
Do we need "Município da Cidade Capital São João Batista"?

Reply:
I kept it because it is the official name, but I guess it could be deleted since most people would probably now more the name in Spanish than the official name.

Reply:
I though I don' need it because "São João" can matched the full name.